### ISSUE TITLE
no SIRI data after 2022/12/27/12/05

### ISSUE THREAD

**Issue by evyatark** [hasadna/open-bus-pipelines#6]
It seems that the latest snapshot in the database is 2022/12/27/12/05 (yesterday at 14:05).

when using this URL:
`https://open-bus-stride-api.hasadna.org.il/siri_snapshots/list?limit=10&get_count=false&snapshot_id_prefix=2022%2F12%2F27%2F12%2F05&order_by=id%20asc`
I get only 1 result (the one from 12:05, GMT+0).

Was there some crash at that time that prevents SIRI data from later times to arrive?

**Comment by OriHoch** [comment]
yes, the MOT SIRI endpoint stopped working.. I emailed them about it
